packages/cli/src/commands/generate/page/__tests__/fixtures/paramPage.js

```diff
@@ -0,0 +1,17 @@
+import { Link, routes } from '@redwoodjs/router'
+
+const PostPage = ({ id }) => {
+  return (
+    <>
+      <h1>PostPage</h1>
+      <p>Find me in "./web/src/pages/PostPage/PostPage.js"</p>
+      <p>
+        My default route is named "post", link to me with `
+        <Link to={routes.post({ id: '42' })}>Post 42</Link>`
```

Comment:
Would a paramPage always have an `id`?

Reply:
I'm not sure I understand what you're getting at.
A paramPage will always have a parameter, but the name is whatever the user wants. For the test that corresponds with this fixture the name of the param is `id`. It comes from https://github.com/redwoodjs/redwood/pull/882/files#diff-6bcf3b847a12bd956cbf4aa88653206eR65

Reply:
Sorry, I was looking at this:
https://github.com/redwoodjs/redwood/pull/882/files#diff-55ff273ce0992ec67f307ed5ba69a541R30-R36

So, the generated param's value would always be `42`? And these values are only used in test and story templates?

Reply:
Yes, the example value will always be `"42"`. I was thinking of maybe making it `"exampleValue"`, or some other string for `{whatever}`, and `42`, for `{whatever:Int}`. But right now the generator isn't that clever. 

It's not used in the story template yet (see my other review comment/question and also the issue about Storybook on Windows). So right now the value is used in the page and test template, but *not* the story template.

Reply:
Ok, awesome! Thanks for the clarification, Maybe we could add a comment for the readers of the code?
`// the 42 values are example values that are used in tests and stories`

Reply:
Comment added. Please let me know if you want me to explain it further in the comment. https://github.com/redwoodjs/redwood/pull/882/commits/a90364b865cc477aecdc62ab5ef149a767687504

Reply:
Thanks, that's perfect!